Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>/node_modules\r\nconfig.js\r\njwtKey.js\r\n.env
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 4c987297165f53e1e5b70a058b62e5c99b31591b)
+++ b/.gitignore	(date 1671116559509)
@@ -1,4 +1,8 @@
 /node_modules
 config.js
 jwtKey.js
-.env
\ No newline at end of file
+.env
+/.idea/.gitignore
+/.idea/modules.xml
+/.idea/topic-backend.iml
+/.idea/vcs.xml
